src/components/Test/Modals/EssayOrSpeakingReportMistake.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
combine EssayOrSpeakingReportMistake.jsx and GrammarOrListeningReportMistake.jsx. You have a lot of repeating code